monest/views.py

```diff
@@ -186,6 +186,12 @@ def company_scores(request):
     return JsonResponse([res], safe=False)
 
 
+@api_view(['GET', 'POST'])
```

Comment:
Hi，small problem here.
1. get rid of 'POST' method in the decorator to inhibit post method
2. get rid of array/list in your response, since you're returning one single object.  Be sure to change the front-end accordingly